packages/statemanager/test/cache.spec.ts

```diff
@@ -2,7 +2,7 @@ import { Trie } from '@ethereumjs/trie'
 import { Account, Address } from '@ethereumjs/util'
 import * as tape from 'tape'
 
-import { Cache } from '../src/cache'
+import { Cache, CacheType } from '../src/cache'
```

Comment:
General question on this, should we be testing with both cache types here (so just run these tests in a loop over eac cache type)?

Reply:
Yes, makes sense, have updated.